### ISSUE TITLE
[Library] Ensure proper encoding of Google Font URL - avoid LinkChecker warning

### ISSUE THREAD

**Issue by richardhand** [adobe/aem-core-wcm-components#711]
Avoid LinkChecker log warnings for Google Font URL:

```
15.08.2019 00:02:45.050 *WARN* [0:0:0:0:0:0:0:1 [1565820164860] GET /content/core-components-examples/library/button.html HTTP/1.1] com.day.cq.rewriter.linkchecker.impl.LinkCheckerImpl Ignoring malformed URI: java.net.URISyntaxException: Illegal character in query at index 46: https://fonts.googleapis.com/
```
by escaping it properly.

**Comment by richardhand** [comment]
Fixed in #702

**Comment by richardhand** [comment]
Code is merged to [master](https://github.com/adobe/aem-core-wcm-components/commits/master) and will be available in the upcoming [2.6.0 Release](https://github.com/adobe/aem-core-wcm-components/milestone/16).
